src/main/webapp/script.js

```diff
@@ -12,6 +12,21 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+/**
+ * Hides reviews when client wantes to see college-info.
+ */
+function hideReviews() {
+  document.getElementById('comment-body').style.visibility = 'hidden';
```

Comment:
Do you want to use `visibility: hidden` for this? That will still cause the elements to take up space, they just won't be visible. Usually we'd want `display: none`.